### DISCUSSION TITLE
cli `remotion render Video out/temp.mp4` this error occurred: `Error while getting compositions`

### DISCUSSION THREAD

**Discussion by winner106** [remotion-dev/remotion#2445] [Q&A] [answered]
Yesterday, everything was functioning properly when I rendered using the CLI. However, today, when I attempted to do it again, I encountered the following error. I haven't made any changes to the code. Do you have any insights into what could be causing this issue? I have already attempted deleting the node_modules directory and clearing the yarn cache for a fresh installation, but these methods have not resolved the problem.

but when I execute  `remotion preview`, everything is fine.

```bash
PS K:\remotion> yarn video
yarn run v1.22.19
$ remotion render Video out/temp.mp4
(1/3) ╭───────────────────╮ Bundled code 3554ms

An error occurred:
Error: Error while getting compositions: Tried to go to http://localhost:3009/index.html and verify that it is a Remotion project by checking if window.getStaticCompositions is defined. However, the function was undefined, which indicates that this is not a valid Remotion project. Please check the URL you passed.
    at innerSetPropsAndEnv (K:\remotion\node_modules\@remotion\renderer\dist\set-props-and-env.js:82:15)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async innerGetCompositions (K:\remotion\node_modules\@remotion\renderer\dist\get-compositions.js:26:5)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

**Comment by winner106** [comment]
@JonnyBurger Do you know what's the reason please?

**Comment by JonnyBurger** [comment, marked as answer]
Hi @winner106!

It appears like there are many dev servers running at once, therefore port 3009 had to be used. While that is okay, it seems like there are other dev servers which might cause a resource conflict.

The best way would be to open `http://localhost:9000` in the browser and see if there is another app running.

Indeed, two apps can use the same port, it's problems like https://github.com/facebook/docusaurus/issues/8202 which can cause this.

**Reply by winner106** [reply]
Thank you, I will check it out

**Reply by winner106** [reply]
I successfully resolved the issue by relocating the entire project folder to a different directory. when executing the CLI command, the following message was displayed:

```bash
🧹 Webpack config change detected. Clearing cache...
```

Since I make frequent modifications to the project code and create different video components every day, it is possible that the issue with webpack caching arises due to the repetitive packaging process. Could you please clarify how to clear the webpack cache in the Remotion project?

**Reply by JonnyBurger** [reply]
Cool!

This should not happen, so concerning (maybe a  Webpack bug) and should be investigated if we can reproduce.

You can clear the cache using `--bundle-cache=false` https://www.remotion.dev/docs/cli/render#--bundle-cache

**Comment by Yobb17** [comment]
https://github.com/orgs/remotion-dev/projects/7?pane=issue&itemId=28541070
